docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/graphql/using-graphql.md

```diff
@@ -0,0 +1,111 @@
+---
+slug: using-graphql
+title: Accessing Ensembl GraphQL service
+description: How to access the GraphQL services provided by Ensembl
+related_articles:
+  - href: ./getting-started.md
+tags:
+  - api
+  - tools
+status: draft
+---
+# Using the Ensembl GraphQL service
+
+The Ensembl GraphQL service can be used to access information about Genes, Transcripts, Assemblies and associated Metadata held by Ensembl.  
```

Comment:
The capitalisation of genes, transcripts, assemblies, and metadata make sense in the context of CDM; but this document is outside the CDM. Why the capitalisation here?